app/javascript/components/search/controls/FacetsPanel.js

```diff
@@ -17,16 +21,66 @@ export default function FacetsPanel() {
   const searchFacetContext = useContext(SearchFacetContext)
   const defaultFacets = searchFacetContext.facets.filter(facet => defaultFacetIds.includes(facet.id))
   const moreFacets = searchFacetContext.facets.filter(facet => moreFacetIds.includes(facet.id))
+  const [showSearchHelpModal, setShowSearchHelpModal] = useState(false)
+
+  const publicStudies = window.SCP.studyStats.public
+  const compliantStudies = window.SCP.studyStats.compliant
+  const percentage = (compliantStudies / publicStudies * 100).toFixed(0)
+  const helpModalContent = (<div>
+    <h4 className="text-center">Advanced Search</h4><br/>
+    Single Cell Portal supports searching on specific facets of studies by ontology classifications.  This allows users
+    to search for studies using metadata values that may not appear in titles or descriptions.
+    <br/><br/>
+    For example, you can search on studies that
+    have <b>species</b> of <b>&quot;Homo sapiens&quot;</b> or have an <b>organ</b> of <b>&quot;brain&quot;</b>.{' '}
+    ~<b>{percentage}% ({compliantStudies} of {publicStudies})</b> public studies in SCP provide this metadata
+    information.
+    {/*
+   more information on public/compliant studies available at
+   https://docs.google.com/spreadsheets/d/1FSpP2XTrG9FqAqD9X-BHxkCZae9vxZA3cQLow8mn-bk
+*/}
+    <br/><br/>
+    For more detailed information, visit
+    our{' '}
+    <a href="https://singlecell.zendesk.com/hc/en-us/articles/360061006431-Search-Studies"
+       target="_blank" rel="noreferrer">documentation
+    </a>.  Study authors looking to make their studies more accessible can read our
+    <a href="https://singlecell.zendesk.com/hc/en-us/articles/4406379107355-Metadata-powered-Advanced-Search"
+       target="_blank" rel="noreferrer"> metadata guide
+    </a>.
+  </div>)
+
+  const advancedOptsLink = <a className="action advanced-opts"
+                              onClick={() => setShowSearchHelpModal(true)}
+                              data-analytics-name="search-help">
+    <FontAwesomeIcon icon={faQuestionCircle} />
+  </a>
+
   return (
-    <>
-      <CombinedFacetControl controlDisplayName="cell type" facetIds={['cell_type', 'cell_type__custom']}/>
+    <div>
+      <span
+        className='badge metadata-search search-title'
+      >
+        Data search { advancedOptsLink }
```

Comment:
> I don't feel too strongly about this -- the typo is my only blocker -- but thought it'd be worth consideration.

The comments about sentence- vs. title-case are warranted, and I'll incorporate them.  As far as look & feel goes, I'll hold off until this can be demo'd tomorrow as getting all the feedback at once will be much simpler than iteratively like this.

Reply:
Also, the term `Data search` was suggested by a stakeholder as an alternative to `metadata` as the worry was that term was not transparent enough.  I don't have a vested interest in one over the other - just consensus.